src/lib/client.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiService } from './api/api.service';
+import { AuthResource } from './resources/auth.resource';
+
+export interface PermanentConstructorConfigI {
+  sessionToken: string;
+  mfaToken: string;
+  accountId: number;
+  archiveId: number;
+}
+
+export class Permanent {
+  private apiKey = 'Xr$k?fopgA"FdWFoPKmmh6n7';
```

Comment:
It's a real API key to the extent that the Permanent system has real API keys, but it's not a _unique_ API key. It's the same client key that the web apps have used since day one, and should probably already have been rotated out anyway. But since there's no special permissions or access granted to this or any API key, and this key has already been in the wild via anyone peeking at outgoing requests their dev tools, it doesn't feel like a critical issue imo.

Now I do believe I meant to not actually commit it, for all the reasons and best practices you mentioned, but accidentally left it in 😅 or maybe I talked myself into leaving it since it's a useless key anyway? Who could know, it was eons ago!

Reply:
Just to chime in, I hear you that there's nothing special about that key at this point, but there *could* be in future, right?  I mean, it won't happen, but in a year is it possible that someone might grant permission to that key?  It's a compromised key, but most of the time people assume keys are not compromised, so we're relying on institutional memory to mitigate that compromise (whereas rotating it out would just elminate the risk).

That said, agreed that it could be prioritized down the line.

Security aside, things like this should be loaded from a configurable variable rather than hard coding for all the reasons Jason mentioned!  +1 to that request.

Reply:
100% onboard, making the change!  I initially didn't want to require an extra configuration param that's going to be identical for all users BUT as mentioned, even if security isn't the main issue, it is a good choice overall to move to a configurable option.